src/test/scala/io/iohk/ethereum/vm/OpCodeGasSpec.scala

```diff
@@ -313,17 +316,17 @@ class OpCodeGasSpec extends FunSuite with OpCodeTesting with Matchers with Prope
       val stateOut = op.execute(stateIn)
 
       val (Seq(_, offset, _, size), _) = stateIn.stack.pop(4)
-      val memCost = calcMemCost(stateIn.memory.size, offset, size)
+      val memCost = config.calcMemCost(stateIn.memory.size, offset, size)
       val copyCost = G_copy * wordsForBytes(size)
-      val expectedGas = G_extcode + memCost + copyCost
+      val expectedGas = G_extcode + memCost + copyCost + 123
```

Comment:
So I was wondering about this `123` and realised that this tests flawed. The reason is that when the execution goes OOG then `verifyGas` only checks that less than `expectedGas` was used.

The fix for now is to change `maxGas` to something like: 

```
val maxGas = 2 * (G_extcode + G_copy * 8)
```

and remove `123` of course.

Unfortunately there may more tests flawed like that. I'll stick a task in to revise this spec later.